app/gui2/src/stores/graph/graphDatabase.ts

```diff
@@ -521,7 +521,7 @@ const baseMockNode = {
   conditionalPorts: new Set(),
 } satisfies Partial<Node>
 
-function mathodCallEquals(a: MethodCall | undefined, b: MethodCall | undefined): boolean {
+export function mathodCallEquals(a: MethodCall | undefined, b: MethodCall | undefined): boolean {
```

Comment:
Why export? This doesn't seem to be used anywhere else.

Reply:
Yeah, but I still think it can become useful in the future, so I don’t want to remove it. export is a way of disabling `unused` linter warning.